romancal/datamodels/dynamicdq.py

```diff
@@ -25,7 +27,8 @@ def dynamic_mask(input_model):
             try:
                 standard_bitvalue = dqflags.pixel[dqname]
             except KeyError:
-                log.warning('Keyword %s does not correspond to an existing DQ mnemonic, so will be ignored' % (dqname))
+                log.warning('Keyword %s does not correspond to an existing DQ mnemonic, '
+                            'so will be ignored' % dqname)
```

Comment:
Should we change keyword to something else? DQ flag?

Reply:
dqflags is already used, so I think qdname should stay as is.